src/Symfony/Component/Messenger/Command/ConsumeMessagesCommand.php

```diff
@@ -61,6 +63,10 @@ protected function configure()
 Use the --limit option to limit the number of messages received:
 
     <info>php %command.full_name% <receiver-name> --limit=10</info>
+
+Use the --memory option to limit the memory consumed by the worker:
+
+    <info>php %command.full_name% <receiver-name> --memory=128</info>
```

Comment:
Would personally call it `--memory-limit` to be the same as in php. It's also not exactly clear that it's in M. Would it be an idea to use `G, M, K` support here, just like with php?

Reply:
Good idea, I did it